docs/README.md

```diff
@@ -62,13 +62,13 @@ npm run start
 
 ## CI Build Flow
 
-The docs are built and published in Github Actions with the `docs.yml` workflow. On each PR, the docs are built, but not published.
+The docs are built and published in GitHub Actions with the `docs.yml` workflow. On each PR, the docs are built but not published.
 
 In each post-commit build, docs are built and published using `vercel` to their respective domain depending on the build branch.
 
-- Master branch docs are published to `edge.docs.solanalabs.com`
-- Beta branch docs are published to `beta.docs.solanalabs.com`
-- Latest release tag docs are published to `docs.solanalabs.com`
+- Master branch docs are published to `edge.docs.anza.xyz`
+- Beta branch docs are published to `beta.docs.anza.xyz`
+- Latest release tag docs are published to `docs.anza.xyz`
```

Comment:
Good to change to make, but separate from the stated mission of "update docs with build from source instructions". Would be happy to stamp this if broken out

Reply:
I guess I can, but feels kinda overkill for a docs PR :shrug:

Reply:
There were other references to `solanalabs.com` URL's so decided to get them all in one go with: https://github.com/anza-xyz/agave/pull/7505